cdc/cdc/kv/region_worker.go

```diff
@@ -730,6 +730,12 @@ func (w *regionWorker) handleResolvedTs(
 		return nil
 	}
 	regionID := state.sri.verID.GetID()
+
+	// In TiKV, when a leader transfer occurs, the old leader may send the last
+	// resolved ts, which may be larger than the new leader appends key to ts.
+	// So we fallback the resolved ts to a safe interval to make sure it's correct.
```

Comment:
```suggestion
	// So we fallback the resolved ts to a safe interval to make sure it's correct.
    // See https://github.com/tikv/migration/issues/193.
    // TODO: fix the issue completely.
```

Also suggest to write a full analysis in #193, to help us discuss with others, and completely fix it later.

Reply:
OK